### PR TITLE
Introduce specific functions for Base64 encoder and decoder

### PR DESCRIPTION
**Pull request by arukshani**
Introduce specific functions for Base64Encoder and decoder

### CODE REVIEW

File: stdlib/ballerina-builtin/src/main/ballerina/jwt/issuer.bal [hunk 1]
```diff
@@ -49,7 +49,9 @@ function createHeader (Header header) returns (string) {
     headerJson[ALG] = header.alg;
     headerJson[TYP] = "JWT";
     headerJson = addMapToJson(headerJson, header.customClaims);
-    return urlEncode(util:base64Encode(headerJson.toString()));
+    string headerValInString = headerJson.toString()  but { () => ""};
+    string encodedPayload =check util:base64EncodeString(headerValInString);
```
Comment (sanjiva) [inline stdlib/ballerina-builtin/src/main/ballerina/jwt/issuer.bal]: Please put a space before "check"

File: stdlib/ballerina-builtin/src/main/ballerina/jwt/issuer.bal [hunk 2]
```diff
@@ -67,20 +69,8 @@ function createPayload (Payload payload) returns (string|error) {
...
-        string returnString => {
-            string encodedString = returnString.replaceAll("\\+", "-");
-            encodedString = encodedString.replaceAll("/", "_");
-            return encodedString;
-        }
-        blob returnBlob => return "error";
-        io:ByteChannel returnChannel => return "error";
-        util:Base64EncodeError returnError => return "error";
-    }
+    string payloadInString = payloadJson.toString()  but { () => ""};
```
Comment (sanjiva) [inline stdlib/ballerina-builtin/src/main/ballerina/jwt/issuer.bal]: Please remove extra space before "but" :-).

File: stdlib/ballerina-builtin/src/main/ballerina/jwt/issuer.bal (hunk 1 above)
Comment (arukshani) [reply stdlib/ballerina-builtin/src/main/ballerina/jwt/issuer.bal]: Fixed

File: stdlib/ballerina-builtin/src/main/ballerina/jwt/issuer.bal (hunk 2 above)
Comment (arukshani) [reply stdlib/ballerina-builtin/src/main/ballerina/jwt/issuer.bal]: Done.

**Pull request merged**